doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -128,3 +128,11 @@ Make sure you ran the following commands to rebuild the classes, objectBricks, f
 ```bash
 bin/console doctrine:migration:exec 'Pimcore\Bundle\CoreBundle\Migrations\Version20230412105530'
 ```
+
+
+### Update Symfony Messenger Worker Configuration
+With Pimcore 11, certain symfony messenger tasks were moved to a separate receiver. Make sure, you adapt your symfony messenger worker configuration (e.g. your supervisor configuration [here](https://github.com/pimcore/skeleton/blob/11.x/.docker/supervisord.conf#LL5C39-L5C90) so that all receivers are consumed. Newly added consumers are: 
```

Comment:
```suggestion
With Pimcore 11, certain Symfony Messenger tasks were moved to a separate receiver. Make sure you adapt your Symfony Messenger worker configuration (e.g. your supervisor configuration [here](https://github.com/pimcore/skeleton/blob/11.x/.docker/supervisord.conf#LL5C39-L5C90) so that all receivers are consumed. Newly added consumers are: 
```